lib/firehose/rack/consumer/http_long_poll.rb

```diff
@@ -124,17 +131,20 @@ def handle_request(request, env)
             # Get the Last Message Sequence from the query string.
             # Ideally we'd use an HTTP header, but android devices don't let us
             # set any HTTP headers for CORS requests.
-            last_sequence = request.params['last_message_sequence'].to_i
+            params = request.params
+            last_sequence = params.delete('last_message_sequence').to_i
```

Comment:
It looks like there might have been a miscommunication. I thought we had settled on [a separate namespace for the params](https://github.com/firehoseio/firehose/pull/67/files#r68436244).

But it looks like we've got `last_message_sequence` and custom params sharing a namespace when it comes to single requests (i.e. the query string). https://github.com/firehoseio/firehose/pull/67/files#diff-701c346719fd241e1617c8584eb8a1ebR17

Do you think this difference is a good idea? To be "consistent" we'd have something like this:
```
/my/channel?last_message_sequence=1&params=foo%3Dbar%26whisky%3D4all
```
That is less readable because you need to escape everything in params.